framework/wazuh/core/cdb_list.py

```diff
@@ -227,7 +227,7 @@ def validate_cdb_list(content: str):
     WazuhError(1112)
         Empty CDB list.
     """
-    regex_cdb = re.compile(r'(?:^"([\w\-:]+?)"|^[^:"\s]+):(?:"([\w\-:]*?)"$|[^:\"]*$)')
+    regex_cdb = re.compile(r'(?:^("|\')([\w\- :]+?)("|\')|^[^:"\s]+):(?:("|\')([\w\- :]*?)("|\')$|[^:\"]*$)')
```

Comment:
This regex looks like this:
![image](https://github.com/wazuh/wazuh/assets/23361101/091f6b6b-28ee-4ee8-932d-6e9d475cfbe2)


It would allow users to set different types of quotes (single/double) at the beginning and the end:
![image](https://github.com/wazuh/wazuh/assets/23361101/319e786d-afa7-4ad5-992b-61da28efd21f)


I would keep the quotes as they were until now, and only add the spaces:
```suggestion
    regex_cdb = re.compile(r'(?:^"([\w\-: ]+?)"|^[^:"\s]+):(?:"([\w\-: ]*?)"$|[^:\"]*$)')
```

Although we could try to support both types of quotes, since there is little left to change the engine, I prefer to include the minimum possible changes.